### PR TITLE
(qtox) Update AU script and change to an embedded package

### PR DESCRIPTION
**Pull request by pascalberger**
Fixes the error in the AU script and change to an embedded package

### CODE REVIEW

File: automatic/qtox/qtox.nuspec [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <metadata>
     <id>qtox</id>
     <title>qTox</title>
-    <version>1.4.1.1</version>
+    <version>1.7.0</version>
```
Comment (AdmiringWorm) [inline automatic/qtox/qtox.nuspec]: please don't change the version to anything newer than what is available on chocolatey.org.
Latest version on chocolatey.org is `1.5.1`

**Comment by AdmiringWorm** [comment]
also had one question, what if the changelog have been updated but the executable haven't yet been built?
We would then package the wrong version.

File: automatic/qtox/qtox.nuspec (hunk 1 above)
Comment (pascalberger) [inline automatic/qtox/qtox.nuspec]: Thanks, missed to reset after last test :). Have it now set to 0.0

**Comment by pascalberger** [comment]
@AdmiringWorm Yes, thats a little bit ugly with determing the version from a different source than the binaries, but unfortunately I couldn't find a better solution. They don't seem to push the Windows Installer any longer to the GitHub releases. Happy to change that if you have a better idea :)

**Comment by AdmiringWorm** [comment]
@pascalberger hmm, perhaps after downloading the executable check the version if it?
If it isn't somewhat equal to the expected version, throw an exception.

File: automatic/qtox/update.ps1 [hunk 2]
```diff
@@ -1,34 +1,72 @@
 ﻿import-module au
```
Comment (AdmiringWorm) [inline automatic/qtox/update.ps1]: We need an import of the chocolateyInstaller.psm1 file here.
i.e.
`Import-Module "$env:ChocolateyInstall\helpers\chocolateyInstaller.psm1"`

**Comment by pascalberger** [comment]
@AdmiringWorm The version of the downloaded installer unfortunately doesn't correspond to the version of the application (seems to always be at 1.0.0)

**Comment by AdmiringWorm** [comment]
@pascalberger I see, so that's a no go then.
Too bad. well, after you've added the missing import module, we'll just merge this.

File: automatic/qtox/update.ps1 (hunk 2 above)
Comment (pascalberger) [inline automatic/qtox/update.ps1]: Ah yes. Added it

**Comment by pascalberger** [comment]
@AdmiringWorm Added the missing import

**Pull request merged**